### PR TITLE
fix:nacos & polaris discovery

### PR DESCRIPTION
**Pull request by chuntaojun**
## Checklist

- [ ] Coverage from `Codecov Report` should not decrease (which means adding junit tests for adding or updating code).
- [ ] All junit tests passing.
- [ ] Add copyright holder at the beginning of .java file if it is new.
- [ ] Add information of this PR to CHANGELOG.md in root of project.

## Checklist (Optional)

- [ ] Will Pull Request to branch of 2020.0 and hoxton.
- [ ] Add documentation in javadoc or comment below the PR if necessary.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/spring-cloud-tencent?pullRequest=816) <br/>All committers have signed the CLA.

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]
Plz add PR info to CHANGELOG.md.

File: spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/DiscoveryUtil.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	}
+
+	/**
+	 * rewrite serviceId when open double registry and discovery by nacos and polaris.
+	 *
+	 * @param serviceId service id
+	 * @return new service id
+	 */
+	public static String rewriteServiceId(String serviceId) {
+		String enableNacos = ApplicationContextAwareUtils.getProperties("spring.cloud.nacos.enabled");
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/DiscoveryUtil.java]: 这些字段是不是可以不用每次都从context获取？第一次获取的时候就存起来？

Comment (chuntaojun) [reply spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/DiscoveryUtil.java]: ok

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]
更新到最新版本polaris-java

**Review by SkyeBeFreeman — APPROVED** [review]

**Pull request merged**